release-notes.md

```diff
@@ -1,6 +1,22 @@
-# Service Broker API Release Notes #
-
-## <a id='2-12'></a>v2.12 ##
+# Service Broker API Release Notes
+
+## v2.13
```

Comment:
I know we don't have this for previous releases, but links from the feature to the relevant part of the spec would be very helpful. I can do this post merge.

Reply:
done